db/migrate/20160429150155_alter_games_table_change_board_column_to_json.rb

```diff
@@ -0,0 +1,6 @@
+class AlterGamesTableChangeBoardColumnToJson < ActiveRecord::Migration
+  def change
+    remove_column(:games, :board)
+    add_column(:games, :board, :json)
```

Comment:
cool, make sure this works, i'm not 100% on the syntax =) after that feel free to merge.